src/OrbitDB.js

```diff
@@ -353,7 +369,10 @@ class OrbitDB {
     // Parse the database address
     const dbAddress = OrbitDBAddress.parse(address)
 
-    if (!options.cache) options.cache = this.caches['default']
+    options.cache = this.caches[options.directory] || this.caches['default']
+    if (options.cache._store.db.status !== 'open') {
+      await options.cache._store.open()
```

Comment:
Same question as in https://github.com/orbitdb/orbit-db/pull/677/files?file-filters%5B%5D=.js#r321580114.

Reply:
Same answer :)